src/lib/event_mapper/aws_event_mapper.py

```diff
@@ -0,0 +1,40 @@
+from .impl import (
+    GeneralAwsEventMapper,
+    GlueEventMapper,
+    StepFunctionsEventMapper,
+    EventParsingException,
+)
+from ..settings import Settings
+
+
+class AwsEventMapper:
+    """Class designed to map AWS events to notification messages"""
```

Comment:
Not for this PR, but for the future let's make the Registry design pattern in a more common way across different components (notification service and alerting parser). 
In notification service we have it a bit differently - 2 methods: register & get (which returns a class object, not a fuction result). 
I think it is a good point for our next brainshtorming :)

Reply:
Yeah, the intention here was to have a single entrypoint for mapping all types of AWS events, so that the lambda wouldn't know anything about the internal mapping logic/providers/registers/etc., but would just call _mapper.to_notification_messages(event)_ instead. And inside, we would have this _AwsEventMapper_ on top of _event_mapper_ package, which would do the event analysis and choosing the appropriate mapper from the _.impl_ (implementation) package, and pass the event there. And there we would have a general mapper, which contains common logic, and specific per AWS service mappers which do their specific parts of the process.

I agree that naming could be a bit ambiguous here (_aws_event_mapper_, _general_aws_event_mapper_), that's why I created an _.impl_ package to mark these modules non-usable directly in the library client code. I just didn't really want lambda to have to call additional _get_mapper_ function to get the appropriate mapper for the event. "Just do the mapping job, aws event mapper!" - that's the intention I wanted lambdas to have :) 

Please advise if we should stick fully to the approach in notification lambda (I agree that concepts alignment for similar solutions is important and makes the whole code a lot more maintainable), or do the renaming part only for now, or should we proceed just like that